spec/forms/additional_claimants_form_spec.rb

```diff
@@ -44,7 +44,7 @@
 
     it 'decorates AdditionalClaimants as Claimants' do
       additional_claimants_form.collection.each_with_index do |_c, i|
-        expect(additional_claimants_form.collection[i].target).to eq collection[i]
+        expect(additional_claimants_form.collection[i].target.as_json).to eq collection[i].as_json
```

Comment:
For some reason, in rails 5, comparing AR objects as equal did not work even if the attributes were the same, so now comparing the results of to_json so the test is simpler